hw_9.py

```diff
@@ -0,0 +1,105 @@
+"""Homework_9"""
+
+# # # Строки с заданным символом # # #
+# Напишите программу, которая бы работала следующим образом -
+# находила символ "#" и если этот символ найден -
+# удаляла предыдущий символ из строки.
+# Ваша задача обработать строки с "#" символом.
+
+
+def string_modifier(string: str) -> str:
+    """Функция замены символа перед #"""
+    if "".join(string.split("#")) == "":
```

Comment:
```suggestion
    if not "#" in string:
```

Reply:
`    if not "#" in string:`
It does not work in case of string "abcde"

Reply:
Just return the string

Reply:
fixed